resources/i18n/hi_HI/cura.po

```diff
@@ -0,0 +1,7748 @@
+msgctxt "@info:question"
+msgid "Are you sure you want to start a new project? This will clear the build plate and any unsaved settings."
+msgstr "क्या आप वाकई एक नया प्रोजेक्ट शुरू करना चाहते हैं? यह बिल्ड प्लेट और सभी न सहेजी गई सेटिंग्स को साफ़ कर देगा।"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/MainWindow/MainWindowHeader.qml:90
+msgctxt "@action:button"
+msgid "Marketplace"
+msgstr "मार्केटप्लेस"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/ConfigurationMenu/AutoConfiguration.qml:18
+msgctxt "@header"
+msgid "Configurations"
+msgstr "कॉन्फ़िगरेशंस"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/ConfigurationMenu/ConfigurationItem.qml:137
+msgctxt "@label"
+msgid "This configuration is not available because %1 is not recognized. Please visit %2 to download the correct material profile."
+msgstr "यह कॉन्फ़िगरेशन उपलब्ध नहीं है क्योंकि% 1 पहचाना नहीं गया है। सही सामग्री प्रोफ़ाइल डाउनलोड करने के लिए कृपया% 2 पर जाएँ।"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/ConfigurationMenu/ConfigurationItem.qml:138
+msgctxt "@label"
+msgid "Marketplace"
+msgstr "मार्केटप्लेस"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/ConfigurationMenu/ConfigurationListView.qml:57
+msgctxt "@label"
+msgid "Loading available configurations from the printer..."
+msgstr "प्रिंटर से उपलब्ध कॉन्फ़िगरेशन लोड हो रहा है..."
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/ConfigurationMenu/ConfigurationListView.qml:58
+msgctxt "@label"
+msgid "The configurations are not available because the printer is disconnected."
+msgstr "कॉन्फ़िगरेशन उपलब्ध नहीं है क्योंकि प्रिंटर डिस्कनेक्ट हो गया है।"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/ConfigurationMenu/ConfigurationMenu.qml:112
+msgctxt "@label"
+msgid "Select configuration"
+msgstr "कॉन्फ़िगरेशन चुनें"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/ConfigurationMenu/ConfigurationMenu.qml:223
+msgctxt "@label"
+msgid "Configurations"
+msgstr "कॉन्फ़िगरेशंस"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/ConfigurationMenu/CustomConfiguration.qml:25
+msgctxt "@header"
+msgid "Custom"
+msgstr "कस्टम"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/ConfigurationMenu/CustomConfiguration.qml:61
+msgctxt "@label"
+msgid "Printer"
+msgstr "प्रिंटर"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/ConfigurationMenu/CustomConfiguration.qml:213
+msgctxt "@label"
+msgid "Enabled"
+msgstr "सक्रिय"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/ConfigurationMenu/CustomConfiguration.qml:267
+msgctxt "@label"
+msgid "Material"
+msgstr "मटेरियलl"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/ConfigurationMenu/CustomConfiguration.qml:407
+msgctxt "@label"
+msgid "इस मटेरियल के चिपकने के साथ बेहतर चिपकाव के लिए गोंद का प्रयोग करें।"
+msgstr "S touto kombinací materiálu pro lepší adhezi použijte lepidlo."
```

Comment:
You changed the msgid here, causing the text to no longer properly be matched...
```suggestion
msgid "Use glue for better adhesion with this material combination."
msgstr "इस मटेरियल के चिपकने के साथ बेहतर चिपकाव के लिए गोंद का प्रयोग करें।"
```